feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/transaction/filter/TransactionFilter.kt

```diff
@@ -0,0 +1,11 @@
+package jp.co.soramitsu.feature_wallet_impl.presentation.transaction.filter
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+class TransactionFilter(
```

Comment:
Why do we need this model? As far as I understood, state sharing is done via TransactionFilterProvider